app/lib/reducers/ui.js

```diff
@@ -24,7 +24,15 @@ const initial = Immutable({
   fontFamily: 'Menlo, "DejaVu Sans Mono", "Lucida Console", monospace',
   fontSizeOverride: null,
   css: '',
-  termCSS: '',
+  termCSS: '' +
```

Comment:
I recommend you instead add it [here](https://github.com/zeit/hyperterm/blob/master/app/lib/components/term.js#L105), since it's technically part of our built-in styles, and this is user-set.